src/pages/signup/index.js

```diff
@@ -47,82 +48,126 @@ const Signup = () => {
-          <label htmlFor="email">{t('signup.labels.email')}</label>
-          <Input
-            register={register}
-            type="email"
-            name="email"
-            error={errors.email}
-            handleFocus={handleFocus}
-          />
+      {!isSend ? (
+        <div className="form column left-column">
+          <form onSubmit={handleSubmit(onSubmit)} noValidate>
+            <h1>{t('signup.title')}</h1>
+            <label htmlFor="username">{t('signup.labels.name')}</label>
+            <Input
+              register={register}
+              type="text"
+              name="username"
+              error={errors.username}
+              handleFocus={handleFocus}
+            />
+            <label htmlFor="email">{t('signup.labels.email')}</label>
+            <Input
+              register={register}
+              type="email"
+              name="email"
+              error={errors.email}
+              handleFocus={handleFocus}
+            />
 
-          <label htmlFor="password">{t('signup.labels.password')}</label>
-          <Input
-            register={register}
-            placeholder={t('signup.labels.passwordPlaceholder ')}
-            type="password"
-            name="password"
-            error={errors.password}
-            handleFocus={handleFocus}
-          />
+            <label htmlFor="password">{t('signup.labels.password')}</label>
+            <Input
+              register={register}
+              placeholder={t('signup.labels.passwordPlaceholder')}
+              type="password"
+              name="password"
+              error={errors.password}
+              handleFocus={handleFocus}
+            />
 
-          <label htmlFor="password">{t('signup.labels.passwordConfirmation')}</label>
-          <Input
-            register={register}
-            type="password"
-            name="passwordConfirmation"
-            error={errors.passwordConfirmation}
-            handleFocus={handleFocus}
-          />
-          <label htmlFor="gender">{t('signup.labels.gender')}</label>
-          <Select
-            name="gender"
-            options={[...GENDER_OPTIONS]}
-            register={register}
-            placeholder={t('signup.labels.selectGender')}
-            error={errors.gender}
-          />
+            <label htmlFor="password">{t('signup.labels.passwordConfirmation')}</label>
+            <Input
+              register={register}
+              type="password"
+              name="passwordConfirmation"
+              error={errors.passwordConfirmation}
+              handleFocus={handleFocus}
+            />
+            <label htmlFor="gender">{t('signup.labels.gender')}</label>
+            <Select
+              name="gender"
+              options={[...GENDER_OPTIONS]}
+              register={register}
+              placeholder={t('signup.labels.selectGender')}
+              error={errors.gender}
+            />
+
+            {error && error.data && (
```

Comment:
Here you can use conditional chaining to access data prop: `error?.data` and omit the first condition